File: rust/examples/voices.rs
mod utils;
use std::error::Error;
use utils::get_tls_options;

use aristech_tts_client::{get_client, get_voices};

#[tokio::main]
async fn main() -> Result<(), Box<dyn Error>> {
    // Load environment variables from .env file
    dotenv::dotenv()?;

    let host = std::env::var("HOST")?;
    let tls_options = get_tls_options()?;
    let mut client = get_client(host, tls_options).await?;

    let voices = get_voices(&mut client, None).await?;
    for voice in voices {
        println!("{:?}", voice);
    }
    Ok(())
}

File: python/examples/voices.py
from aristech_tts_client import TtsClient
from utils import host, auth_token, auth_secret, root_cert, ssl

client = TtsClient(host=host, ssl=ssl, root_cert=root_cert, auth_token=auth_token, auth_secret=auth_secret)
voices = client.list_voices()
for voice in voices:
    print(voice)
